widgetssdk/src/main/java/com/glia/widgets/webbrowser/WebBrowserActivity.kt

```diff
@@ -7,7 +7,14 @@ import android.os.Bundle
 import com.glia.widgets.base.FadeTransitionActivity
 import com.glia.widgets.databinding.WebBrowserAvtivityBinding
 
-
+/**
+ * Glia internal class.
+ *
+ * It will be automatically added to the integrator's manifest file by the manifest merger during compilation.
+ *
+ * This activity is used to display simple web pages. For example, it can be used to display links to the client's Terms and Conditions or
```

Comment:
Is this line length a problem to be fixed, or rather not?

Reply:
We don't have a strict policy on line length but I think it makes sense to fix it here